nexus/db-queries/src/db/raw_query_builder.rs

```diff
@@ -88,14 +88,14 @@ type BoxedQuery = diesel::query_builder::BoxedSqlQuery<
 //
 // But this relies on nightly features.
 pub struct QueryBuilder {
-    query: BoxedQuery,
+    query: Option<BoxedQuery>,
```

Comment:
To make the `RawPaginator` API with the interface that exists, these functions needed to act on `&mut self` instead of `self`.

This was a breaking change, and it requires using an `Option<BoxQuery>` internally, but that inner type detail should not be visible to callers.